hedera-node/hedera-app/src/main/java/module-info.java

```diff
@@ -42,6 +42,9 @@
     requires com.swirlds.merkle;
     requires com.swirlds.merkledb;
     requires com.swirlds.virtualmap;
+    requires com.fasterxml.jackson.core;
```

Comment:
Why can't we use PBJ instead? We should try to avoid new dependencies.

Reply:
I have discussed this with @rbair23 and @jasperpotts. We should use PBJ and JSON instead.